test/passing/list_pattern.ml

```diff
@@ -4,3 +4,42 @@ let _ =
   match x with Atom x -> x | List (Atom x :: Atom y :: rest) -> x ^ y
 
 let _ = match x with (x :: y) :: z -> true
+
+let x = function
+  | [ "Lorem ipsum dolor sit amet, consectetur adipiscing elit"
+    ; [ "Lorem ipsum dolor sit amet, consectetur adipiscing elit"
+      ; (* ", sed do eiusmod tempor incididunt ut labore et dolore"; *)
+        "sed do eiusmod tempor incididunt ut labore et dolore"
+        (* " magna aliqua. Ut enim ad minim veniam, quis nostrud "; *)
+        (* "exercitation ullamco laboris nisi ut aliquip ex ea commodo " *)
+       ]
+    ; (* ", sed do eiusmod tempor incididunt ut labore et dolore"; *)
+      "sed do eiusmod tempor incididunt ut labore et dolore"
+      (* " magna aliqua. Ut enim ad minim veniam, quis nostrud "; *)
+      (* "exercitation ullamco laboris nisi ut aliquip ex ea commodo " *)
+     ] ->
```

Comment:
Unecessary space here and above

Reply:
Yes, didn't find a simple way to fix that, but at least it's not a regression caused by this PR (already in issue77.ml for example) and I don't think it's directly linked to this issue so we could fix it in another PR (better to keep the two diff's apart).